internal/kube_helper.go

```diff
@@ -0,0 +1,169 @@
+package internal
```

Comment:
Create a kube package in internal.. Going forward this functions will grow and will be hard to manage in one package

Reply:
This is the only thing we need. I don't think internal package will be flooded with changes. If needed in the future, it's easier to refactor and create new package.